src/routes/app/+page.ts

```diff
@@ -1,11 +1,30 @@
-import { search } from "$lib/api/documents.js";
+import type { SearchOptions } from "$lib/api/types";
+import { search } from "$lib/api/documents";
 
 export async function load({ url, fetch }) {
   const query = url.searchParams.get("q") || "";
-  const searchResults = search(query, true, fetch);
+  const per_page = +url.searchParams.get("per_page") || 25;
```

Comment:
```suggestion
  const per_page = +url.searchParams.get("per_page") || DEFAULT_PER_PAGE;
```
Saving the default as a constant someplace (`$lib/constants.ts`?) is a good idea, especially since we'll have similar options in other load functions.

Reply:
This might go away entirely. The `per_page` was useful for pagination, but now it's not really exposed. I was using it locally since I don't have many documents, and this let me test loading. 

If we do want to have a constant, I'd put it in `src/config/config.js`.